### ISSUE TITLE
GRAPHITE_FROM does not effect actual graphite queries

### ISSUE THREAD

**Issue by erez-rabih** [arachnys/cabot#401]
I pulled master after a few months I haven't and now the GRAPHITE_FROM does not effect the from parameter on the graphite render requests.

Any workaround for this?

**Comment by erez-rabih** [comment]
Just to update this - I went back to commit 5819473c31aeb3da1ced3b1f3c9e461bcdfaf7b8 and the GRAPHITE_FROM works again.

I guess some of the latest commits broke this behaviour.

**Comment by JeanFred** [comment]
Thanks for reporting and investigating this @erez-rabih. Looks like the commit is 27a353a.

**Comment by JeanFred** [comment]
So what happens is that if `frequency` in the Graphite check is filled in, the `GRAPHITE_FROM` will not be used.

**Comment by erez-rabih** [comment]
Makes sense because I did see a correlation between the from parameter and the check frequency.

**Comment by erez-rabih** [comment]
@JeanFred  It would also be nice to have the ability to override this value for a specific check

**Comment by erez-rabih** [comment]
@JeanFred I think we should not set a dependency between the check frequency and the timerange we're looking at on graphite.
Maybe these could be two different settings per check?
